po/zh_CN.po

```diff
@@ -0,0 +1,552 @@
+# Copyright (C) 2016 Mario Cianciolo <mr.udda@gmail.com>
+# This file is distributed under the same license as the multiload-ng package.
+# Roberta Assennato <r.assennato@gmail.com>, 2016.
```

Comment:
There should be your name and email ( `Dee.H.Y <dongfengweixiao@hotmail.com>, 2016` )